### PR TITLE
Fix sha256 for 24.5.0

### PR DESCRIPTION
**Pull request by kenodegard**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Since the build for #225 failed (see https://github.com/conda-forge/conda-build-feedstock/runs/24916644440) I did not bump the build number.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 
 source:
   url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 97938c7940777d014ded9b3031e8ae0bf516e68937ffb4f43075ea5ffccc1d24
+  sha256: 019a2893af9fb2e1aceed4ff66cabff35e0717195f25e90bed0dc1c119b4814a
 
 build:
   number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml:13]: While the previous build never went out, still think it is a good idea to bump the `build/number` to communicate there was an issue encountered with the previous build

```suggestion
  number: 1
```

**Review by jakirkham — COMMENTED** [review]
Thanks Ken! 🙏

Had one suggestion above

Also opened an upstream issue. Though looks like you already put a fix together (thank you!). So linked it to the PR to close it

File: recipe/meta.yaml (hunk 1 above)
Comment (kenodegard) [reply recipe/meta.yaml]: fair enough, either works for me

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Review by jakirkham — APPROVED** [review]

**Pull request merged**